docs/user_guide/08_visualizations.md

```diff
@@ -59,4 +59,3 @@ For results with more than one predicted class, the plot_results function will d
 ```
 color_palette = sv.ColorPalette.from_matplotlib('viridis', 6)
 plot_results(results, ground_truth=ground_truth, results_color=color_palette)
-``
```

Comment:
Looks like this should probably be adding a back tick to make three back ticks not removing the exiting backticks.